types/tizen-common-web/tslint.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules":{
+        "no-const-enum": false,
```

Comment:
This rule is on for a very good reason - when you use a const enum, the const is directly compiled into your code, thus quietly burning a dependency on the version of the enum used at _build_ time into your application or library when used in an expression position (which is not guaranteed to match with the version used at _run_ time). For a final, built application, this is _OK_, but on DT, we can't say if consumers will be building a library or a complete application (that's up to them) - so we have to be very cautious. We offer string literal completions for the union members - can't those be good enough?

In short: Please don't disable this rule. Some of the others I can abide disabling (like line length), at least at first, for a large package like this, but this one creates a big usability issue for type consumers. (Unless those enum objects are actually available at runtime, in which case, normal non-const enums should be used, so the values are "dynamically" selected at runtime)

Reply:
I understood. Please check the latest commit.